lib/config/options/index.ts

```diff
@@ -1891,6 +1931,7 @@ const options: RenovateOptions[] = [
     default: false,
     supportedPlatforms: ['bitbucket'],
     globalOnly: true,
+    inheritConfigSupport: true,
```

Comment:
We should add this to the docs options table

Reply:
Seems it's done automatically